Localizations/KeepingYouAwake/de.xliff

```diff
@@ -154,10 +154,12 @@ Rechtsklick, um das Menü anzuzeigen</target>
       </trans-unit>
       <trans-unit id="Disable menu bar icon highlight color">
         <source>Disable menu bar icon highlight color</source>
+        <target>Auswahlfarbe im Statusmenü nicht anzeigen</target>
```

Comment:
The formulation is still not very specific. It probably should be something like ‘Keine Auswahlfarbe beim Klicken des Statusmenüs anzeigen’.